src/Controller/ActivityController.php

```diff
@@ -61,4 +61,25 @@ public function getActivityDetails(SerializerInterface $serializer, $id): Respon
 
         return new JsonResponse($json, 200, [], true);
     }
+
+    /**
+     * @Rest\Delete("/activities/delete/{id}")
+     * @param int $id
+     * @return Response
+     */
+
+    public function deleteActivity($id): Response
+    {
+        $em = $this->getDoctrine()->getManager();
+        $activity =  $em->getRepository(Activity::class)->find($id);
+
+        if (!$activity) {
+            throw $this->createNotFoundException('No activity found for id '.$id);
+        }
+
+        $em->remove($activity);
```

Comment:
Implement a method `delete()` in the activity repository to remove the activity.